src/components/Post/newPost.js

```diff
@@ -513,7 +508,7 @@ const PostUpload = ({ isOpen, onClose }) => {
                                     <div className="px-4 py-3 bg-gray-50 text-right sm:px-6">
                                         <button
                                             type="button"
-                                            className="inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-gradient-to-r from-blue-500 to-teal-500 text-base font-medium text-white hover:from-pink-500 hover:to-yellow-500 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 sm:text-sm"
+                                            className="inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-gradient-to-r from-blue-500 to-teal-500 text-base font-medium text-white hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 sm:text-sm"
```

Comment:
we require this?

Reply:
i just modified the button hover ness here